### DISCUSSION TITLE
Generate fake serial number for octoprint_deploy

### DISCUSSION THREAD

**Discussion by CruisnGrrl** [mriscoc/Ender3V2S1#1075] [Ideas]
Is it possible for the firmware to generate a serial number for use by octoprint? octoprint_deploy allows the ability to control multiple printers but the lack of serial number on the printer requires that the printer be bound to a particular usb port.

**Comment by mriscoc** [comment]
Yes, you need to do a custom compile of the firmware or subscribe to [Patreon](https://www.patreon.com/mriscoc/membership) to get a official customized version.
